skills/ingest-challenges/SKILL.md

```
@@ -113,15 +113,24 @@ description: 外部ソース（共有 repo / Notion / Google Doc / Slack ログ
+- **フィンガープリント `fp`**: **取得したての外部ソース本文**（`github-issue` なら Issue の `body`）の更新検知用の指紋（人間は読まないので HTML コメントに隠す）。**算出はスクリプトで行う**（LLM の暗算・目視相当も、規約を読んで手で再実装するのも不可）:
+
+  ```bash
+  gh issue view <番号> --repo <owner>/<repo> --json body -q .body \
+    | "${CLAUDE_PLUGIN_ROOT}/scripts/ingest-fp.sh"
+  #=> 12 桁の小文字 16 進（例: 9c56b5c0a92d）
+  ```
+
+  - **入力は外部本文そのものだけ**。台帳に書いた文字列（説明・完了条件・緊急度）は**一切入力にしない**。他 type（`repo-file` / `mcp-doc` / `mcp-chat`）でも同様に、そのソースから取得した当該エントリの本文をそのまま標準入力へ渡す。
+  - **正規化・ハッシュの正本は `${CLAUDE_PLUGIN_ROOT}/scripts/ingest-fp.sh`**（本文冒頭にコメントで規約を明記。テストは `scripts/tests/ingest-fp.test.sh` がゴールデン値で固定）。本ドキュメントは**算式を再掲しない**——再掲すると読み手が写しから手で再実装し、実装と乖離する。**この節を読んで自分でハッシュを組み立てない**（手で再実装して 4 回誤った実績がある）。
+  - **`fp` は台帳だけからは再計算・監査できない**（外部ソースへの到達が要る）。これは意図した設計上の性質であり、監査は上の 1 コマンドを当該エントリの外部本文へ再実行して行う。外部が close / 削除されて本文を取得できない場合は**監査不能**であって「一致」でも「不一致」でもない——`fp` を書き換えず、判定を推測で埋めない。なお通常の取り込みではこの状況は判定に影響しない（本文を取得できないエントリはそもそも手順2の候補に入らず、この照合に到達しない）。
+  - **fp 不一致でも、取得した本文と台帳の説明欄（`（原文引用）` 直下のブロック引用から `> ` を外したもの）を直接比較して実質同一なら**（過去の算出方法の揺れ等）、人間記入欄・取り込み日は更新せず **fp のみ現行の算出方法で更新**する。
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)